spec/features/node_pages/notes_table_toolbar_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+describe 'node pages' do
+  describe '#show notes table toolbar', js: true do
+    subject { page }
+
+    before do
+      login_to_project_as_user
+
+      @node  = create(:node)
+      @notes = []
+      (Note::MAX_DELETED_INLINE + 1).times do |i|
+        @notes << create(:note, node: @node, text: "#[Title]#\r\ntest#{i}\r\n\r\n#[Description]#\r\nnote#{i}\r\n")
+      end
+
+      visit node_path(@node, tab: 'notes-tab')
+    end
+
+    context 'when Select All is clicked' do
```

Comment:
some of these are very similar to the specs in `spec/features/issue_pages/table_toolbar_spec.rb`. Maybe it would be better to add shared examples?

Reply:
👍